### PR TITLE
Upgrade to Admin UI 1.20.0

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

@proddata made some fixes and improvements to the Admin UI [1]. This patch will bring in the most recent release.

With kind regards,
Andreas.

[1] https://github.com/crate/crate-admin/blob/main/CHANGES.rst#2021-09-27-1200

### CODE REVIEW

**Review by proddata — COMMENTED** [review]
🙏

File: docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -141,4 +144,7 @@ Fixes
 
 - Fixed an issue that caused showing an incorrect log message in case of an
   authentication failure. "Password authentication" used to be shown instead
-  of the actual authentication method name.
\ No newline at end of file
+  of the actual authentication method name.
+
+- Fixed an issue in the administration console: Nested arrays in
```
Comment (mfussenegger) [inline docs/appendices/release-notes/unreleased.rst:149]: We shouldn't have fixes within a feature release.

Would it be possible to do a 1.19.2 with the fix - backport that to CrateDB 4.6, and remove the entry from this PR?

Comment (amotl) [reply docs/appendices/release-notes/unreleased.rst]: Thanks. I just removed this entry from the PR and will hand in another PR only including this fix, for targeting the 4.6 series.

**Review by mfussenegger — APPROVED** [review]

File: docs/appendices/release-notes/unreleased.rst (hunk 1 above)
Comment (amotl) [reply docs/appendices/release-notes/unreleased.rst]: See #11776.

**Comment by amotl** [comment]
Retest this please.

**Pull request merged**